tests/unit/executors/indexers/test_binary_indexer.py

```diff
@@ -47,10 +47,10 @@ def test_binarypb_update1(test_metas, delete_on_dump):
     save_abspath = idxer.save_abspath
 
     with BaseIndexer.load(save_abspath) as idxer:
-        assert idxer.query('1') == b'oldvalue'
+        assert idxer.query('1') == [b'oldvalue']
```

Comment:
```suggestion
        assert idxer.query(['1']) == [b'oldvalue']
```

Reply:
Otherwise the test as u write it, works only because a `string` can `iterate` on characters. But like this it is much more clear